src/ErrorSparseTask.cpp

```diff
@@ -114,14 +114,14 @@ void ErrorSparseTask::run(const Configuration& config) {
   std::cout << "Compute error task connecting to store" << std::endl;
 
   std::cout << "Creating sequential S3Iterator" << std::endl;
-
+  s3_initialize_aws();
   uint32_t left, right;
   if (config.get_model_type() == Configuration::LOGISTICREGRESSION) {
     left = config.get_test_range().first;
```

Comment:
The Collaborative Filtering uses the train set to test (I'm not sure why), so, since get_train_range now returns a vector due to the cross_validation changes (the collaborative filtering train range will be a vector with 1 element), we must get the first element from the vector in order to test.